src/components/OverviewCard.jsx

```diff
@@ -11,9 +23,131 @@ import {
   retrieveResourceFileMetadata,
 } from '../utils';
 
+// axios settings
+axios.defaults.withCredentials = true
+
 const OverviewCard = ({
   date, category, settingsToggle, itemIndex, siteName, fileName, isResource, isHomepage
 }) => {
+  const dropdownRef = useRef(null)
+  const fileMoveDropdownRef = useRef(null)
+  const categoryInputRef = useRef(null)
+  const [allCategories, setAllCategories] = useState()
+  const [newCategory, setNewCategory] = useState('')
+  const [errorMessage, setErrorMessage] = useState('')
+  const [canShowDropdown, setCanShowDropdown] = useState(false)
+  const [canShowFileMoveDropdown, setCanShowFileMoveDropdown] = useState(false)
+  const [canShowDeleteWarningModal, setCanShowDeleteWarningModal] = useState(false)
+  const baseApiUrl = `${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}${category ? isResource ? `/resources/${category}` : `/collections/${category}` : ''}`
+
+  useEffect(() => {
+    const fetchData = async () => {
+      // Retrieve the list of all page/resource categories for use in the dropdown options.
+      if (isResource) {
+        const resourcesResp = await axios.get(`${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/resources`);
+        const { resources: allCategories } = resourcesResp.data;
+        setAllCategories(allCategories.map((category) => category.dirName))
+      } else {
+        const collectionsResp = await axios.get(`${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/collections`);
+        const { collections: collectionCategories } = collectionsResp.data;
+        setAllCategories(collectionCategories)
+      }
+    }
+    fetchData()
+  }, [])
+
+  useEffect(() => {
+    if (canShowFileMoveDropdown) fileMoveDropdownRef.current.focus()
```

Comment:
Focus isn't lost when we open up the `ComponentSettingsModal` - you can see the dropdown in the background, which only disappears when we click on something in the modal:

<img width="1075" alt="Screenshot 2020-11-16 at 5 45 57 AM" src="https://user-images.githubusercontent.com/31984694/99197713-10e3a500-27cf-11eb-9075-b02eda82b1e8.png">

The `ComponentSettingsModal` should behave in the same way, so that focus is automatically set on it when it's opened

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/219/commits/55304db5869db66976b522d7adc71ed23f432155